### PR TITLE
Third party docs

### PR DESCRIPTION
**Pull request by nporter-adbe**
Adds docs for partners and customers using third party extensions.

### CODE REVIEW

File: Documentation/ThirdParties/Customers.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Third Party Extension Support for Customers
+
+All existing third party extensions are compatible with the AEP Swift SDK. Using third-party extensions will need to use the backward compatibility layer and update how extensions are registered. This document will walk through the steps to get setup using third party extensions in the AEP Swift SDK.
+
+## Quick Setup
+
+#### Install Compatibility Layer
+
+Third party extensions requires the backward-compatible layer. Instructions on installing the backward compatible layer can be found [here](./Migration/ACP-Migration.md).
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Customers.md:9]: `Existing Third party extensions require the backwards-compatibility layer.`

I think this is more accurate, right?

File: Documentation/ThirdParties/Customers.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+#### Update Extension Registration
+
+Now that the backward compatible layer is installed, `ACPCore` can now be imported into the `AppDelegate` file allowing third party extensions to be registered. Replace usage of `MobileCore.registerExtensions` with `ACPCore.registerExtensions` while including the additional third party extensions.
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Customers.md:13]: `ACPCore` can ~now~ be imported...

File: Documentation/ThirdParties/Customers.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Third Party Extension Support for Customers
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Customers.md:1]: `Third party` if you're using it as a noun, `third-party` if you're using it as an adjective.  probably should be the later for most instances on this page.

File: Documentation/ThirdParties/Customers.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Now that the backward compatible layer is installed, `ACPCore` can now be imported into the `AppDelegate` file allowing third party extensions to be registered. Replace usage of `MobileCore.registerExtensions` with `ACPCore.registerExtensions` while including the additional third party extensions.
+
+```diff
++ import ThirdPartyExtension
+
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
++  // Register any third party extensions before `registerExtensions`
++  ThirdPartyExtension.registerExtension()
+   MobileCore.registerExtensions([Lifecycle.self, Identity.self, Signal.self, ...], {
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Customers.md:22]: per your previous comment, this should be `ACPCore.registerExtensions`, right?

File: Documentation/ThirdParties/Partners.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Third Party Extension Support for Partners
+
+Existing third party extensions are compatible with the AEP Swift SDK without making any changes. This is made possible by the [backward compatible layer](./Migration/ACP-Migration.md). Any customer who is consuming a legacy third party extension is required to use the backward compatible layer. We encourage existing partners to test their legacy extensions with the AEP Swift SDK to ensure the extension is behaving as expected.
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Partners.md:3]: `backwards compatibility layer`

**Review by sbenedicadb — CHANGES_REQUESTED** [review]
just a few small grammatical things that need to be addressed.  

can we also update the documentation landing page README to help make the docs for third-party support more discoverable?

**Comment by nporter-adbe** [comment]
@sbenedicadb I'll be making another PR with an update to our README/table of contents to make other docs more discoverable.

File: Documentation/ThirdParties/Customers.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# Third party Extension Support for Customers
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Customers.md:1]: `Third-party`

File: Documentation/ThirdParties/Partners.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Third party Extension Support for Partners
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Partners.md:1]: `Third-party`

File: Documentation/ThirdParties/Partners.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
...
+
+Existing third-party extensions are compatible with the AEP Swift SDK without making any changes. This is made possible by the [backward compatible layer](./Migration/ACP-Migration.md). Any customer who is consuming a legacy third party extension is required to use the backwards compatibility layer. We encourage existing partners to test their legacy extensions with the AEP Swift SDK to ensure the extension behaves as expected.
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/Partners.md:3]: `Third-party`

File: Documentation/ThirdParties/README.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Third Party Extension Support
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/README.md:1]: `Third-party`

File: Documentation/ThirdParties/README.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+
+All third party extensions built on the previous generation of the Adobe Experience Platform will be compatible with the latest AEP Swift SDK. Using existing third party extensions in AEP Swift can be achieved by using a compatibility layer to bridge between the legacy ACPCore and the new AEPCore. The following documents guide partners who have developed an extension and for customers who are currently using third party extensions.
```
Comment (sbenedicadb) [inline Documentation/ThirdParties/README.md:3]: fourth-party? 😝

**Pull request merged**